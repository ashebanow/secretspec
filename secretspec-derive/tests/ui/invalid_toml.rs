use secretspec_derive::define_secrets;

// This should fail because the TOML is invalid
define_secrets!("tests/fixtures/invalid_toml.txt");

fn main() {}