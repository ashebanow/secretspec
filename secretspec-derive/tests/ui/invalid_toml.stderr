error: Failed to read invalid_toml.txt: No such file or directory (os error 2)
 --> tests/ui/invalid_toml.rs:4:1
  |
4 | define_secrets!("invalid_toml.txt");
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `define_secrets` (in Nightly builds, run with -Z macro-backtrace for more info)
